llvm/lib/Target/EraVM/AsmParser/EraVMAsmParser.cpp

```diff
@@ -289,9 +289,6 @@ bool EraVMAsmParser::MatchAndEmitInstruction(SMLoc Loc, unsigned &Opcode,
 // Auto-generated by TableGen
 static unsigned MatchRegisterName(StringRef Name);
 static unsigned MatchRegisterAltName(StringRef Name);
-static void applyMnemonicAliases(StringRef &Mnemonic,
```

Comment:
Maybe keep the declaration and call? So if we'd add some aliases in the future they will apply automatically?

Reply:
Unfortunately, it seems `applyMnemonicAliases` is only generated if at least one alias exists. Maybe defining a dummy weak function would help, but I doubt it works in the same translation unit.

Reply:
Ah, ok, makes sense. Ok, then!